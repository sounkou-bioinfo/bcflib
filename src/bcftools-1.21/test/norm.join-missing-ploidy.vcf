##fileformat=VCFv4.1
##fileDate=20190708
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=chr1,length=248956422>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SAMPLE_ONE	SAMPLE_TWO
chr1	100	.	A	T	.	.	.	GT	0/1	./.
chr1	100	.	A	C	.	.	.	GT	.	1
chr1	200	.	A	C	.	.	.	GT	.	1
chr1	200	.	A	T	.	.	.	GT	0/1	./.
