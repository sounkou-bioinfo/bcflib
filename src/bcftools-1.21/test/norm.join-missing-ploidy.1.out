##fileformat=VCFv4.1
##FILTER=<ID=PASS,Description="All filters passed">
##fileDate=20190708
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
##contig=<ID=chr1,length=248956422>
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	SAMPLE_ONE	SAMPLE_TWO
chr1	100	.	A	T,C	.	.	.	GT	0/1	2
chr1	200	.	A	C,T	.	.	.	GT	0/2	1
